packages/grid/src/__tests__/__e2e__/Grid.cy.tsx

```diff
@@ -409,90 +409,40 @@ describe("Grid", () => {
     });
   });
 
-  describe("Row Selection", () => {
-    describe("Uncontrolled & switching selection modes", () => {
-      it("Shows correct columns", () => {
```

Comment:
Do we not have to test this anymore?

Reply:
We do - these tests that have been removed were simply repeated. You could see them present on lines 
https://github.com/jpmorganchase/salt-ds/pull/1141/files/37d58c0e9cda7534ed9120f85997d6a60993dc68#diff-70083ed0f96d8fa99e5060f265de64108b3cfaa8922bdd71dc2f16a900015665R458
and
https://github.com/jpmorganchase/salt-ds/pull/1141/files/37d58c0e9cda7534ed9120f85997d6a60993dc68#diff-70083ed0f96d8fa99e5060f265de64108b3cfaa8922bdd71dc2f16a900015665R413